src/styles.css

```diff
@@ -144,6 +144,33 @@ notifier-container ul{
 .pointer {
     cursor: pointer;
 }
+.listIconColor {
+    color: #399cca
+}
+.cardTopRow {
+    background-color: white;
+}
+.cardHeadband {
+    background-color: #399cca;
+    padding: .25rem;
+    color: white;
+    font-family: Lato;
+    font-size: 1.3rem;
+}
+.deleteListButton {
```

Comment:
This should be in the component that is using it. Not a concern for the entire application